### ISSUE TITLE
"Cannot read property 'Workspace' of undefined" after upgrading to Angular v11

### ISSUE THREAD

**Issue by TonySpegel** [angular-schule/angular-cli-ghpages#119]
Hi after upgrading Angular to v11 I've got this error:

```bash
> ng deploy --base-href=/test-project/ --no-silent

❌ An error occurred when trying to deploy:
Cannot read property 'Workspace' of undefined
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! test-project@0.0.0 deploy: `ng deploy --base-href=/test-project/ --no-silent`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the test-project@0.0.0 deploy script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```
I'm sorry if  isn't related to the update but it has been working before that

**Comment by JohannesHoppe** [comment]
see #118, we are working on a fix. Please stay tuned. 👍

**Comment by TonySpegel** [comment]
oh my bad, I'm sorry. Haven't seen this.

**Comment by JohannesHoppe** [comment]
No problem at all. Let's keep it open so that other people are aware, too.

**Comment by Githamza** [comment]
Same here :/

**Comment by JohannesHoppe** [comment]
I'm actively working on a fix, which includes a breaking change. So I would like to deliver a solution that will make the transition as easy as possible. See PR #120 

If you can't wait, then please use the standalone CLI version of angular-cli-ghpages. This version works without any dependencies to Angular:
https://www.github.com/angular-schule/angular-cli-ghpages/tree/master/docs%2FREADME_standalone.md

**Comment by JohannesHoppe** [comment]
**Thanks for the patience!**

angular-cli-ghpages v1.0.0-rc.1 with support for Angular v11 is out. Please give it a try and report any issues here!

Fixed via #118 and #120.
Release notes: https://github.com/angular-schule/angular-cli-ghpages/releases/tag/v1.0.0-rc.1

----

Please note the **⚠️ BREAKING CHANGE (v1)**

Starting with version 1 the option `--configuration` was renamed to `--build-target`.

BEFORE (_does not work_):

```
ng deploy --configuration=test
```

NOW:

```
ng deploy --build-target=test
```

If you use the old syntax, you will probably receive the following error:

> An unhandled exception occurred: Configuration 'test' is not set in the workspace.
